CMSgov/onemac
# oy2-9173: adapt styles for split screen

Story: https://qmacbis.atlassian.net/browse/OY2-9173
Endpoint: https://d3d5le6pfngzg5.cloudfront.net/

### Details

Update styles to support viewports of 768px and upward.

### Changes

- Modify home page styles to stack gracefully and make large logo disappear on "medium" screens
- Modify FAQ contact info box per [the current Figma](https://www.figma.com/file/HKT5yDoWteNfLzVPDkJOIl/State-Users?node-id=9%3A3260)
- Tweak styles on ChoiceList component to flex more gracefully
- Improve stacking behavior in footer
- Modify user page to stack

### Implementation Notes

🤷 

### Test Plan

1. Open the home page and resize the window to 768px, 1024px, and 1280px. Verify that it is styled to handle the various sizes without loss of grace or functionality.
2. Login as `helpdeskactive` and visit the following pages, repeating the procedure from step 1 on each:
    - Dashboard
    - a Submission Details page
    - User Management
    - User Page
    - FAQ
3. Log out, then login as `statesubmitteractive`. Repeat step 2 with the following pages:
    - New Submission menu
    - New SPA menu
    - New Waiver menu
    - Submission Form
